src/Nethermind/Nethermind.Blockchain/ISyncConfig.cs

```diff
@@ -29,6 +29,9 @@ public interface ISyncConfig : IConfig
         [ConfigItem(Description = "If set to 'true' then the Fast Sync (eth/63) synchronization algorithm will be used.", DefaultValue = "false")]
         bool FastSync { get; set; }
         
+        [ConfigItem(Description = "Relevant only if 'FastSync' is 'true'. If set to value, then it will set a minimum height threshold limit up to witch Full Sync will stay on when chain will be behind network.", DefaultValue = "null")]
+        long? FastSyncCatchUpHeightDelta { get; set; }
```

Comment:
FastSyncTrigger / FastSyncTriggeringLag /FullSyncStickiness / BlocksBehindToFastSync

Reply:
HeightDelta might not be too bad

Reply:
maybe just change documentation and leave the name this way?